lib/byron/src/Cardano/Wallet/Byron/Network.hs

```diff
@@ -326,9 +326,10 @@ mkWalletClient
         -- ^ Static blockchain parameters
     -> TQueue m (ChainSyncCmd ByronBlock m)
         -- ^ Communication channel with the ChainSync client
-    -> NetworkClient m
-mkWalletClient bp chainSyncQ =
-    nodeToClientProtocols NodeToClientProtocols
+    -> m (NetworkClient m)
+mkWalletClient bp chainSyncQ = do
+    stash <- atomically newTQueue
```

Comment:
This could maybe be called "responseBuffer"

Reply:
done in: 0446953c8f6a888a6af5d4d19895d8b918a5884f :+1: